src/frontends/sql/decoder_context_test.cc

```diff
@@ -31,14 +31,23 @@ using testing::UnorderedElementsAre;
 using testing::Values;
 
 using ir::Attribute;
+using ir::Block;
 using ir::IRContext;
 using ir::Operation;
 using ir::Storage;
 using ir::Value;
 using ir::types::TypeBase;
 using ir::value::Any;
-using ir::value::StoredValue;
 using ir::value::OperationResult;
+using ir::value::StoredValue;
+
+TEST(DecoderContextTest, BuildTopLevelBlock) {
+  IRContext context;
+  DecoderContext decoder_context(context);
+
+  const Block &block = decoder_context.BuildTopLevelBlock();
+  EXPECT_EQ(block.module(), &decoder_context.global_module());
```

Comment:
Didn't we change `block.module()` to `block.parent_module()`?

Reply:
Yes. I changed the field but forgot the getter :(

Reply:
I'll do a follow-on PR to fix that.